useragent.go

```diff
@@ -167,6 +173,15 @@ func (ua *UserAgent) Platform() string {
 
 }
 
+func (ua *UserAgent) IsIos() bool {
+	platform := ua.Platform()
+	return platform == Iphone || platform == Ipad || platform == Ipod
+}
+
+func (ua *UserAgent) IsRemind() bool {
```

Comment:
I'd say we either need to make prefix configurable or make the repo private at this point